application.go

```diff
@@ -60,6 +61,7 @@ type ListApplicationsResponse struct {
 	Links Links `json:"_links"`
 }
 
+// List the applications on the Nexmo account
```

Comment:
The [convention](https://blog.golang.org/godoc-documenting-go-code) for the first sentence of a Godoc docstring is that the first word should be the method/struct/whatever being described, so I think this _should_ be:

`// ListApplications lists the applications on the Nexmo account`

Given that you've done a great job of fleshing out the required docs, I'm going to accept what you've provided, but I'll do some research to see how many projects out there are actually following those conventions to the letter.